Core.DomainModel/ItSystemUsage/Read/ItSystemUsageOverviewRoleAssignmentReadModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Core.DomainModel.ItSystemUsage.Read
+{
+    public class ItSystemUsageOverviewRoleAssignmentReadModel : IHasId
+    {
+        public int Id { get; set; }
+
+        public int RoleId { get; set; }
+
+        public int UserId { get; set; }
+
+        public string UserFullName { get; set; }
+
+        public int ParentId { get; set; }
+
+        public virtual ItSystemUsageOverviewReadModel Parent { get; set; }
```

Comment:
Husk at blænde den af i odata config.. ingen grund til at den fremgår af skemaet også selvom det er en read model den peger på

Reply:
![image](https://user-images.githubusercontent.com/39791946/114710528-1d443d80-9d2e-11eb-976e-3bd87c63b4af.png)

Jeg kan ikke få lov at blænde af for den